### PR TITLE
Enable proxy and OAuth in Vectra

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
In Hold(Verifying bugfix with customers)

## Related Issues
related to: demisto/etc#15183
related to: demisto/etc#14414

## Description
Enabled use of system proxy as well as OAuth tokens.

## Does it break backward compatibility?
   - Yes
       - Further details: Basic auth has been deprecated and tokens are the preferred auth method.

## Must have
- [ ] Tests (We don't have an instance)
- [x] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by amshamah419** [comment]
@GavrielFilippov @AndyDemisto Since we do not have an instance, could you guys have the customer verify that the integration works for them? If we have confirmation, we can merge these changes.

File: Integrations/integration-Vectra.yml [hunk 1]
```diff
@@ -56,20 +63,20 @@ script:
...
-        'vectra-health':'/api/health/',
-        'vec-get-detetctions-by-id':'/api/detections/', //deprecated
-        'vec-detections': '/api/detections/',   //deprecated
-        'vec-hosts':'/api/hosts',               //deprecated
-        'vec-settings':'/api/settings',         //deprecated
-        'vec-triage':'/api/rules',              //deprecated
-        'vec-sensors':'/api/sensors',           //deprecated
-        'vec-health':'/api/health/',            //deprecated
-        'vec-get-host-by-id':'/api/hosts/',     //deprecate
+        'vectra-detections': '/api/v2/detections/',
```
Comment (yuvalbenshalom) [inline Integrations/integration-Vectra.yml]: why are we using v2 api here? is that connected to the fix?

**Comment by Itay4** [comment]
@yuvalbenshalom whats our status here?

**Comment by amshamah419** [comment]
@Itay4 @yuvalbenshalom We may need to close out this PR. We tried doing the simple fixes with the customer, but there seems to be a major discrepancy between V1 and V2. Since we don't have an instance, we are stalled at the moment.

**Comment by amshamah419** [comment]
Going to close out this PR since we really need an instance to get the fixes done.

**Pull request closed without merging**